src/includes/properties/class-papi-property-term.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+// Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Papi Property Term.
+ *
+ * @package Papi
+ */
+class Papi_Property_Term extends Papi_Property {
+
+	/**
+	 * The default value.
+	 *
+	 * @var null
+	 * @since 1.0.0
+	 */
+
```

Comment:
I have moved away from new line after comment block so this should be fixed everywhere.